dist/logs/logzioLogsFunction/core.js

```diff
@@ -1,60 +1,186 @@
+    fs.mkdir(newFolderName, { recursive: true }, err => {
+        if (err) throw err;
+    });
+    return newFolderName;
+};
+let folderName = createFolderName();
+let fileName = '';
+const workingDir = process.cwd();
+const createFileName = (context) => {
+    fileName = 'logs-'.concat(uniqString()).concat('.txt');
+    return fileName;
+};
+const maxBatchSize = 100;
+const MB = 1000;
+let folderSize = 0;
+let logIndex = 1;
+const isObject = obj => {
+    return obj !== undefined && obj !== null && obj.constructor == Object;
+};
+const deleteEmptyFields = obj => {
+    for (const key in obj) {
+        if (isObject(obj[key])) {
+            deleteEmptyFields(obj[key]);
+        }
+        if (key === '') {
+            delete obj[key];
+        }
+    }
+};
+const getCallBackFunction = context => {
+    return function callback(err, bulk) {
+        if (err) {
+            context.err(`logzio-logger error: ${err}`, err);
+        }
+        context.log('Done.');
+        context.done();
+    };
+};
+const getParserOptions = () => {
+    return {
+        token: process.env.LogzioLogsToken,
+        host: process.env.LogzioLogsHost,
+        bufferSize: Number(process.env.BufferSize),
+        timeout: Number(process.env.Timeout),
+    };
+};
+const getContainerDetails = () => {
+    return {
+        storageConnectionString: process.env.LogsStorageConnectionString,
+        containerName: containerName,
+    };
+};
+const deleteDirsWithFiles = (foldersToDelete, context) => {
+    foldersToDelete.forEach(folderPath => {
+        fs.rmdirSync(folderPath, { recursive: true });
+    })
 }
 
-const getParserOptions = () => ({
-  metrics: {
-    token: process.env.LogzioMetricsToken,
-    host: process.env.LogzioMetricsHost,
-  },
-  logs: {
-    token: process.env.LogzioLogsToken,
-    host: process.env.LogzioLogsHost,
-  },
-});
-
-module.exports = function processEventHubMessages(context, eventHubMessages, {
-  enableMetric = false,
-}) {
-  const {
-    host,
-    token,
-  } = enableMetric ? getParserOptions().metrics : getParserOptions().logs;
-  context.log(`Starting Logz.io Azure function with enableMetrics: ${enableMetric}`);
-  context.log(JSON.stringify(eventHubMessages));
-  const callBackFunction = getCallBackFunction(context);
-
-  const logzioShipper = logger.createLogger({
-    token,
-    host,
-    type: 'eventHub',
-    protocol: 'https',
-    internalLogger: context,
-    compress: true,
-    debug: true,
-    callback: callBackFunction,
-  });
-
-  const dataParser = new DataParser({
-    internalLogger: context,
-    enableMetric,
-  });
+const uploadFiles = async (containerClient, filesToUpload, context) => {
+    try {
+        const promises = [];
+        for (const file of filesToUpload) {
+            const blockBlobClient = containerClient.getBlockBlobClient(file);
+            promises.push(blockBlobClient.uploadFile(file));
+        }
+        await Promise.all(promises);
+        if (!arrayIsEmpty(filesToUpload)) {
```

Comment:
```suggestion
        if (filesToUpload.length !== 0)) {
                context.log('Uploaded logs to back up container.');
        } 
```
and remove the `isArrayEmpty` func